### DISCUSSION TITLE
why is protable version build with mingw but the installer installs msvc version

### DISCUSSION THREAD

**Discussion by CodeCraftPlugin** [PrismLauncher/PrismLauncher#2879] [Q&A] [answered]
For some reason the portable version is MinGW build from the website but the installer installs msvc build

**Comment by Ryex** [comment, marked as answer]
Because MSVC builds link with the MSVC c++ runtime and can thus never be portable

**Reply by Kationor** [reply]
And why not use the same MinGW binary for portable and nonportable? Does MSVC offer any significant advantages?

**Reply by Trial97** [reply]
Before 8.4 we had them both using MSVC.
MSCV updated and broke Prism for a lot of people( the old version of MSCV runtime was no longer compatible with the newer ones).
Then we decided to switch the portable to MinGW.
The only advantage of MSCV over MinGW is a smaller installation media and the fact that using system libraries may (sometimes) be better over the libraries shipped with MinGW.
